packages/react-server/core/__tests__/integration/redirectForward/RedirectForwardSpec.js

```diff
@@ -1,12 +1,12 @@
-var helper = require("../specRuntime/testHelper"),
+var helper = require("../../../test/specRuntime/testHelper"),
```

Comment:
How about:
- helpers in a separate directory under packages
- don't publish to npm
- include a dev dependency in `react-server` package.json of 
```javascript
"react-server-test-helpers": "file:../react-server-test-helpers"
```

Then you can require the package like normal, and I __think__ dev dependencies get wiped away when published to npm.